kfs-core/src/main/resources/edu/arizona/kfs/gl/spring-gl.xml

```diff
@@ -21,6 +21,27 @@
                 <value>edu/arizona/kfs/gl/ojb-gl.xml</value>
             </list>
         </property>
+        <property name="jobNames">
+			<list>
+				<value>enterpriseFeedJob</value>
+				<value>collectorJob</value>
+				<value>nightlyOutJob</value>
+				<value>budgetAdjustmentCashTransferJob</value>
+				<value>scrubberJob</value>
+				<value>posterJob</value>
+				<value>posterBalancingJob</value>
+				<value>sufficientFundsAccountUpdateJob</value>
+                <value>sufficientFundsFullRebuildJob</value>

```

Comment:
Changed to list merge="true" and only added the new job name.  Fixed spacing issues.